recipe/migrations/google_cloud_cpp250.yaml

```diff
@@ -1,14 +0,0 @@
-__migrator:
-  build_number: 1
-  kind: version
-  migration_number: 2
-
-# needs to pin down to patch level due to inline namespaces changing, see
-# https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/3430
-google_cloud_cpp:
-- '2.5.0'
-
-libgoogle_cloud:
-- '2.5.0'
```

Comment:
Actually, this should have been `libgoogle_cloud_cpp` - doesn't matter only arrow was affected and we don't use that output (yet...?)

Should still be added to the global pinning in this PR though.

CC @hmaarrfk

Reply:
Thank you for catching this.